common/scaffolding/table-en.html

```diff
@@ -0,0 +1,119 @@
+---
+{
+	"title": "Table - Scaffolding",
+	"language": "en",
+	"altLangPage": "table-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-10-19"
+}
+---
+
+<h2>The <code>table</code> element</h2>
+
+<table></table>
+<pre>
+<code>&lt;table&gt;&lt;/table&gt;</code>
+</pre>
+
+<h3>2 rows & 2 columns with the <code>tr</code> and <code>td</code> elements</h3>
+
+<table>
+<tr>
+	<td>Lorem ipsum dolor sit amet</td>
+	<td>Proin pellentesque sodales cursus</td>
+</tr>
+<tr>
+	<td>Nunc dictum felis at dignissim dictum</td>
+	<td>Curabitur ut leo nisi</td>
+</tr>
+</table>
+<pre>
+<code>&lt;table&gt;
+&lt;tr&gt;
+	&lt;td&gt;Lorem ipsum dolor sit amet&lt;/td&gt;
+	&lt;td&gt;Proin pellentesque sodales cursus&lt;/td&gt;
+&lt;/tr&gt;
+&lt;tr&gt;
+	&lt;td&gt;Nunc dictum felis at dignissim dictum&lt;/td&gt;
+	&lt;td&gt;Curabitur ut leo nisi&lt;/td&gt;
+&lt;/tr&gt;
+&lt;/table&gt;</code>
+</pre>
+
+<h3>The <code>th</code> element defines a header cell</h3>
+
+<table>
+<tr>
+	<th>Lorem ipsum dolor sit amet</th>
+	<th>Proin pellentesque sodales cursus</th>
+</tr>
+<tr>
+	<td>Nunc dictum felis at dignissim dictum</td>
+	<td>Curabitur ut leo nisi</td>
+</tr>
+</table>
+<pre>
+<code>&lt;table&gt;
+&lt;tr&gt;
+	&lt;th&gt;Lorem ipsum dolor sit amet&lt;/th&gt;
+	&lt;th&gt;Proin pellentesque sodales cursus&lt;/th&gt;
+&lt;/tr&gt;
+&lt;tr&gt;
+	&lt;td&gt;Nunc dictum felis at dignissim dictum&lt;/td&gt;
+	&lt;td&gt;Curabitur ut leo nisi&lt;/td&gt;
+&lt;/tr&gt;
+&lt;/table&gt;</code>
+</pre>
+
+<h3>Grouping tabular cells with the <code>thead</code>, <code>tbody</code>, <code>tfoot</code>, <code>colgroup</code> and <code>col</code> elements</h3>
+
+<table>
+<colgroup>
+	<col span="1">
+</colgroup>
```

Comment:
You are missing one `<colgroup>` element because that width must be equal as any row width in the table.

Reply:
Fixed in commit "Fixed errors and added images".